### ISSUE TITLE
Fans always on?? Macbook 15 2019 Big Sur

### ISSUE THREAD

**Issue by DunhamGitHub** [DanielStormApps/Fanny#78]
Are the fans really always!! on on my Macbook 15 2019?

Just booted up (cold chassi) cannot hear any fan, 0% CPU, yet fans on!? <img width="24" src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/apple/285/dizzy-face_1f635.png" style="display:inline-block">

<img width="231" alt="Screen Shot 2021-06-03 at 08 05 38" src="https://user-images.githubusercontent.com/6884752/120570635-c3fdae00-c442-11eb-9698-18a54727d901.png">

**Comment by jeffreytgilbert** [comment]
Yes. They really always are. It's worse if you plug in an external monitor because if you have one with a discrete gpu chip in it, that will then activate, get hotter, and cause them to run even faster. I record audio and am on calls all day and this was a huge issue for my sanity daily. I recently purchased the new M1 Max MBP and have run super compute and gpu intensive tasks and the fans never turned on. At some point they may have run at 24 rpm briefly, but that's imperceptible and every time i look right now it's at 0. You can always tell via the mic noise. It makes the built in mics on the old intel macbooks totally useless, and it's also super annoying background noise even with an external mic. 

I spent a significant amount of my personal time looking for ways to stop the fan noise on the old intel macbooks, but there wasn't a good universal solution beyond just getting an m1 macbook.
